src/pages/MatchList/MatchList.tsx

```diff
@@ -27,14 +19,21 @@ export const MatchList = () => {
   })
 
   return (
-    <motion.div {...animationProps} style={{ width: '100%' }}>
-      <List sx={{ width: 'inherit' }}>{matchItems}</List>
-      <SpeedDial
-        ariaLabel="Create new match"
-        sx={{ ...styledSpeedDial }}
-        icon={<SpeedDialIcon />}
+    <motion.div
+      {...animationProps}
+      style={{ width: '100%', position: 'relative' }}
+    >
+      <Box height="100%" overflow="auto">
+        <List sx={{ paddingBottom: '80px' }}>{matchItems}</List>
+      </Box>
+      <Fab
+        color="primary"
+        aria-label="add"
+        sx={{ ...styledFloatButton }}
         onClick={() => navigate('/create-match')}
```

Comment:
I suggest you to create a file called "routeSpecs" and store all the routes we have there, to avoid repeating those strings everywhere. Let's talk about this, idea for the future